examples/ssx-test-serverless-dynamodb-api/README.md

```diff
@@ -0,0 +1,118 @@
+# SSX Serverless - AWS Node.js Typescript + DynamoDB
+
+## Setup
+
+### Install dependencies
+```bash
+yarn
+# OR
+npm i
+```
+
+### Fill the .env file
+```bash
+cp .env.example .env
+# fill the variables
+```
+
+### Install `serverless` globally
+```bash
+npm install --global serverless
+```
+
+# Run DynamoDB 
+
+## **Using** Docker
+
+```bash
+docker-compose up -d dynamo
+sls dynamodb start --migrate
```

Comment:
If the `INFURA_ID` is empty in the `.env` file this throws.

```
Error:
Cannot resolve serverless.ts: Variables resolution errored with:
  - Cannot resolve variable at "provider.environment.INFURA_ID": Value not found at "env" source
  -
```

Reply:
Yes, it is necessary to follow all the configuration steps in sequence as described in the README. Setting environment variables is mandatory